topologies/kne/ixia-arista-4ports.textproto

```diff
@@ -0,0 +1,89 @@
+name: "ixia-c"
```

Comment:
I think the name "arista_ixia.textproto" is more consistent with the existing "arista_ceos.textproto". @bstoll wdyt?

Reply:
Sounds good to me